arbos/l1pricing/l1pricing.go

```diff
@@ -673,11 +674,15 @@ func makeFakeTxForMessage(message core.Message) *types.Transaction {
 	if gasFeeCap.Sign() == 0 {
 		gasFeeCap = randomGasFeeCap
 	}
+	gas := message.Gas()
+	if gas == 0 {
+		gas = RandomGas
```

Comment:
Curious what was the effect of using gas = 0 in the fake tx prior to this PR? Would this also lead to gas over-estimation?

Reply:
Using '0' would lead to potential underestimation it is much more compressible than random data so the return from gas estimation could have been too low. With this change we might overestimate a bit, but that's much safer